css/style.css

```diff
@@ -0,0 +1,67 @@
+*{
+    box-sizing: border-box;
+    color:white;
+}
+
+body{
+    background-image: linear-gradient( 115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7) ), url(https://www.truck1.ec/img/Maquinaria_de_construccion_Cargadora_de_ruedas_Caterpillar_966_K-xxl-1194/1194_9189449724081.jpg);
+}
```

Comment:
Your feedback is valuable in helping them improve their code. To further enhance your feedback, consider providing specific suggestions for improvement. Here's an example of how you can phrase your feedback:

"Good job with the implementation of classes and styles! I noticed that you haven't used a dot at the beginning of defining every class. To ensure consistency and improve readability, I suggest adding a dot before each class name. Additionally, using 2 spaces for indenting, as recommended by the linters, will help ensure compliance with coding standards. This will make the code more maintainable and easier to read. Keep up the great work and continue your code!"